src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.RuntimeIdentifierInference.targets

```diff
@@ -91,7 +76,13 @@ Copyright (c) .NET Foundation. All rights reserved.
       </PropertyGroup>
     </When>
 
-    <!-- NOTE: PlatformTarget=arm64 is not currently supported and therefore no inference of that here. -->
+    <When Condition="($(RuntimeIdentifier.EndsWith('-arm64')) or $(RuntimeIdentifier.Contains('-arm64-'))) and
+                     '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and '$(_TargetFrameworkVersionWithoutV)' >= '6.0'">
```

Comment:
Is there a reason to restrict this to .NET 6 and higher?  Is this because ARM64 isn't supported in .NET 5, or to avoid breaking compatibility, or what?

Reply:
Good question! I believe ARM64 is fully supported in .NET 5, so we might check for .NET 5 and higher instead. I was afraid of possible compatibility issues. I hit this issue while working on Native AOT for ARM64. I have not seen complains from regular .NET 5 users regarding this.

Reply:
I would lean towards removing the target framework check altogether.  If you try to target ARM64 on .NET Core 3.1 for example, I expect you'd get an error saying that there wasn't a runtime pack available for that RuntimeIdentifier.  So there's no real need to avoid setting the PlatformTarget in such a case.

We could conceivably break projects which were using an ARM64 RuntimeIdentifier but didn't want the PlatformTarget set to Arm64.  That seems pretty unlikely to me though, what do you think?  @rainersigwald, any thoughts?

If we do remove the framework check, then I would also like to retarget this PR to release/5.0.2xx, so it can go in the next feature release of the 5.0 SDK.  That also would make it less likely that it would cause compat breaks, as there would be less time for people to create projects relying on the old behavior.

Reply:
No deep thoughts but I think I agree with your take @dsplaisted.

Reply:
In that case, I would propose we remove the framework check in this condition, take the change for 5.0.2xx, and add this to the list of possibly (but not likely) breaking changes for that release.  Does that sound OK @KathleenDollard @marcpopMSFT?

Reply:
@AntonLapounov I think we're good to take this, but can you retarget it to the release/5.0.2xx branch?

Reply:
@dsplaisted I am going to do it today.